mock/mock.spec

```diff
@@ -35,6 +38,7 @@ Conflicts: mock-core-configs < 32.6
 
 # Requires 'mock-core-configs', or replacement (GitHub PR#544).
 Requires: mock-configs
+Requires: %{name}-filesystem
```

Comment:
I think we should require `%name-fliesystem = %version-%release`.

Reply:
Why? I am not fan of this. If we ever need some functionality from mock-filesystem then we can put there `>= X.Y`. No need to enforce exact version when it is not needed.

Reply:
I thought it is per Fedora guidelines, but [it's just a suggestion, not must](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_requiring_base_package).  And the suggestion is only related to the base package....

If you **do not** overspecify the Requires, it can (in theory at least) cause some troubles on version mismatch in the future.  While there are no consequences when you do so.  Nevermind, I'm not going to require a fix here.

Reply:
> Nevermind, I'm not going to require a fix here.

I still prefer it though... (when both packages come from the same spec).  But feel free to merge as is.